spec/middleware_spec.rb

```diff
@@ -113,6 +138,21 @@ def app
     end
   end
 
+  describe "/tidewave/config" do
+    it "returns JSON configuration" do
+      config.team = { id: "dashbit" }
```

Comment:
I have no idea if writing values here is global and therefore bad

Reply:
https://github.com/tidewave-ai/tidewave_rails/blob/4eeb2f3b372812e7e5ef11e8a153eef82075ddc1/spec/middleware_spec.rb#L9

A new `config` is built for every test, so it's fine!